libc/src/math/generic/ceilf.cpp

```diff
@@ -12,6 +12,12 @@
 
 namespace LIBC_NAMESPACE {
 
-LLVM_LIBC_FUNCTION(float, ceilf, (float x)) { return fputil::ceil(x); }
+LLVM_LIBC_FUNCTION(float, ceilf, (float x)) {
+#ifdef __LIBC_USE_BUILTIN_CEIL_FLOOR_TRUNC
+  return __builtin_ceilf(x);
+#else
+  return fputil::ceil(x);
```

Comment:
Can we not just make `fputil::ceil()` use the builtin? That way internal usage gets the more optimal version.

Reply:
I talked about this with Tue yesterday. We were initially talking about doing this in FPUtil, but yesterday he suggested having the builtin/generic switch in the entrypoints directly because `fputil::{ceil,floor,round,trunc}` are only used in the respective entrypoints (actually `fputill::trunc` which is used in unit tests for `modf`, but nowhere in libc/src/).